obstacles_frames: [marker_1, marker_2]
obstacles_move_probabilities: [0.5, 0.5]